### ISSUE TITLE
JOSS Review : Performance & Example usage

### ISSUE THREAD

**Issue by sepandhaghighi** [silvandeleemput/memcnn#22]
The memory consumption table (table 3) is vague, you should compare memory usage between `MemCNN` vs other platforms.

You mentioned :

Table 3 shows the average memory usage during model training using MemCNN. **The results show significant memory savings using the invertible operations as the number of layers increases for ResNet without the use of invertible operations** (Comparison reference ??)

**Comment by silvandeleemput** [comment]
@sepandhaghighi Hi, I agree with you that the table description and purpose were not very clear. The intent of this table was to compare similar CNN implementations (all in PyTorch) with (RevNet) and without (ResNet) memory savings against each other. Here the RevNet implementations use the MemCNN reversible block. So the comparisons are: ResNet-32 vs RevNet-38, ResNet-110 vs RevNet-110, and ResNet-164 vs RevNet-164. The comparison shows that the use of MemCNN (RevNet implementations) achieves significant memory savings for the activations over conventional implementations (ResNet implementations). 

Additionally, I found the table lacking some important details so I have split the memory usage into parameter memory usage and activations memory usage, which required recomputing the memory usage altogether. Furthermore, I have added the number of parameters for each model to the table as well.

The new statistics have been added to the table and the description has been changed to clarify the table's purpose. These changes affect both the `paper.md` and the `readme.rst`.

**Comment by sepandhaghighi** [comment]
@silvandeleemput Thanks for your efforts, now it's meaningful.
As a final suggestion, I think it will better if you show related items in one row (for example `ResNet-3`2 vs `RevNet-38`) or show `RevNet` and `ResNet` details in separate tables (peer to peer).

 Notice  : For both `Table3` and `Table2`

**Comment by silvandeleemput** [comment]
@sepandhaghighi Good suggestion! I have changed `Table 3` to have a row-wise comparison for `Resnet` vs `RevNet`. Furthermore, I have split `Table 2` into `Table 2a` and `Table 2b` for prediction accuracy and training time respectively and reordered the headers such that the comparison between the Tensorflow versus PyTorch implementations is easier to perform. Overall, these changes should facilitate the intended comparisons. 

Changes have been made to both the `paper.md` and the `readme.rst`.

**Comment by sepandhaghighi** [comment]
Well done!
